src/go/rpk/pkg/cli/topic/config.go

```diff
@@ -66,6 +66,14 @@ valid, but does not apply it.
 			// deletions are just keys.
 			setKVs, err := parseKVs(sets)
 			out.MaybeDie(err, "unable to parse --set: %v", err)
+			// Redpanda fails to set both remote.read and write when passed
+			// at the same time, we err here instead.
+			// See https://github.com/redpanda-data/redpanda/issues/9191
+			_, read := setKVs["redpanda.remote.read"]
+			_, write := setKVs["redpanda.remote.write"]
+			if read && write {
+				out.Die("unable to set both 'redpanda.remote.read' and 'redpanda.remote.write' at the same time. Please set them separately")
+			}
```

Comment:
Two thoughts:
* We don't yet have good guidance on multi-sentence exit messages, but I don't think we should have lowercase for the first sentence, uppercase for the second, and no punctuation on the second. If we want to stick with an error message here, I think I'd change the latter part to `, please set them separately` and avoid any period / second sentence uppercase
* Rather than failing, wdyt about issuing the alter twice?

Reply:
1) ok, will change.

2) Per the issue https://github.com/redpanda-data/redpanda/issues/9191

> or even break it into multiple requests if needed (although that may be OTT).

It might be OTT... I went this way since it is supposedly a quick workaround until redpanda solves the root issue. 

So this is issuing the incrementalAlterConfigRequest of either read/write to every topic and then sending the rest of the changes in another request, right?

Reply:
This is the alternative: https://github.com/redpanda-data/redpanda/compare/4c37cf62f00db2d84f713d8b61ea16a2b3938273..896de4df0450d1b5fbcf2f76d481cd602eae9730

lmk what do you think.